CHANGELOG.md

```diff
@@ -15,6 +15,9 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#1078](https://github.com/realm/SwiftLint/issues/1078)
 
+* Enable extension_access_modifier rule.  
```

Comment:
I think this can be confusing, since it's not something that SwiftLint users care about. Can you remove the entry? 😬